src/rust/engine/graph/src/entry.rs

```diff
@@ -309,12 +309,20 @@ impl<N: Node> Entry<N> {
         match context.graph().dep_generations(entry_id, &context).await {
           Ok(ref dep_generations) if dep_generations == &previous_dep_generations => {
             // Dependencies have not changed: Node is clean.
+            context.stats().cleaning_succeeded += 1;
             true
           }
-          _ => {
+          x => {
```

Comment:
Could we possibly provide a better error message in the trace than the debug representation of this `Result` type?

Reply:
I can't think of one.